src/devices/Mhz19b/Mhz19b.cs

```diff
@@ -0,0 +1,256 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.IO.Ports;
+using System.Text;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Mhz19b
+{
+    /// <summary>
+    /// MH-Z19B CO2 concentration sensor binding
+    /// </summary>
+    public sealed class Mhz19b : IDisposable
+    {
+        private enum Command : byte
+        {
+            ReadCo2Concentration = 0x86,
+            CalibrateZeroPoint = 0x87,
+            CalibrateSpanPoint = 0x88,
+            AutoCalibrationSwitch = 0x79,
+            DetectionRangeSetting = 0x99
+        }
+
+        private enum MessageFormat
+        {
+            Start = 0x00,
+            SensorNum = 0x01,
+            Command = 0x02,
+            DataHighRequest = 0x03,
+            DataLowRequest = 0x04,
+            DataHighResponse = 0x02,
+            DataLowResponse = 0x03,
+            Checksum = 0x08
+        }
+
+        private const int MessageSize = 9;
```

Comment:
`MessageBytes`/`MessageBits` make sure you got unit in the name

Reply:
Good point! Done.